src/styles.scss

```diff
@@ -35,15 +35,22 @@ $Cain_-theme: mat.define-light-theme((
 
 /* You can add global styles to this file, and also import other style files */
 
-body {
-    margin: 0;
-    padding: 0;
-    font-size: 16px; // set rem here for entire application
-}
-
 @font-face {
     font-family: "Inter";
     src: local("Inter"), url(./assets/fonts/Inter/Inter-Regular.ttf) format("truetype");
 }
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+
+* {
+  margin: 0;
```

Comment:
if you wanted to add color: white here instead of in the form.component.html that way it is the default across the app.

Reply:
Not sure what you mean.